src/engine/DrawManager.java

```diff
@@ -440,6 +440,11 @@ public void drawAmmo2(final Screen screen, final int magazine2, final int bullet
 		backBufferGraphics.drawString("BUL: " + Integer.toString(10-bullet_count2) + "/" + Integer.toString(magazine2), 224, screen.getHeight() + 25);
 	}
 
+	public void drawLevel(final Screen screen, final GameState gameState) {
```

Comment:
There is an old method in here from a previous feature. I think it is not used so you should remove it.

Reply:
done